pkg/cmd/init/cmd.go

```diff
@@ -54,6 +54,17 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 		"If set, the command will initialize the OCM control plan in foreground.")
 	cmd.Flags().StringVarP(&o.output, "output", "o", "text", "output foramt, should be json or text")
 	cmd.Flags().BoolVar(&o.singleton, "singleton", false, "If true, deploy singleton controlplane instead of cluster-manager. This is an alpha stage flag.")
+	cmd.Flags().StringVar(&o.operatorResourceQosClass, "operator-resource-qos-class", "Default", "the resource QoS class of the cluster manager operator. Can be one of Default or BestEffort")
```

Comment:
I think we could ref the command here https://kubernetes.io/docs/reference/kubectl/generated/kubectl_set/kubectl_set_resources/.

something like:

clusteradm init --resource-requests=cpu=xxx,memory=xxx --resource-limits=cpu=xxx,memory=xxx

it is not necessary to specify the type, we can decide type if resource-requests is set.

Reply:
I updated the flag value format as you suggested.
type is still necessary I suppose because we still need to know it's Default or BestEffort if limits and requests are not set.